.gitignore

```diff
@@ -3,3 +3,5 @@ lite-engine.exe
 .env
 get-docker.sh
 /release/
+/lite-engine-windows-amd64.exe
```

Comment:
we can summarise ignoring binaries with something like 'lite-engine*'

Reply:
done